contracts/interchain-token-service/src/state.rs

```diff
@@ -48,10 +48,28 @@ pub enum TokenSupply {
 
 /// Information about a token on a specific chain.
 #[cw_serde]
-pub struct TokenChainInfo {
+pub struct TokenInstantiation {
```

Comment:
This name is worse imo, since supply is being updated and not just instantiated

Reply:
changed it to `Instance` (which was the intended name, I got the word wrong 😅 ). I'm open to suggestions though